src/Microsoft.Health.Fhir.Web/appsettings.json

```diff
@@ -50,7 +50,7 @@
     "InitialDatabaseThroughput": null,
     "ConnectionMode": "Direct",
     "ConnectionProtocol": "Tcp",
-    "ContinuationTokenSizeLimitInKb": 2,
+    "ContinuationTokenSizeLimitInKb": 1,
```

Comment:
what's the reason for changing to 1k and also do we need to worry about backwards compatibility? E.g., someone executed a search with 2k as limit and received a continuation token with 2k then the server gets upgraded and the search request comes in with 2k continuation token but now the limit is set to be 1k?

Reply:
We were testing if this was going to resolve problems executing history/paging tests against appservice. The appdeploy.json already sets 1k as the default also.
I'm not sure this makes the tests any more reliable though, thinking of abandoning

Reply:
We actually have this applied on the arm template already, so this change should not be necessary

Reply:
@brendankowitz , @feordin I think I figured out the issue. In CI, this setting was not applied so the continuation token would be longer than app service would accept. App Service then responds with a 404 without forwarding the request to us, which I believe is why we did not see the request in AI.